app/src/main/java/kr/ac/kaist/team888/core/Region.java

```diff
@@ -116,4 +116,47 @@ public float getMaxY() {
   public void setMaxY(float maxY) {
     this.maxY = maxY;
   }
+
+  /**
+   * Do linear transformation on destination region.
+   *
+   * @param dst destination region
+   * @param stroke stroke to transform
+   * @return new stroke on destination region
+   */
+  public Stroke transformStroke(Region dst, Stroke stroke) {
+    Stroke.StrokeBuilder builder = new Stroke.StrokeBuilder()
+        .setStartPoint(transformPoint2D(dst, stroke.getStartPoint()))
+        .setEndPoint(transformPoint2D(dst, stroke.getEndPoint()));
+
+    for (Point2D controlPoint : stroke.getControlPoints()) {
```

Comment:
Suggest using `addControlPoints(ArrayList<Point2D> controlPoints)` on  line185 of /core/Stroke.java.
I think it is more proper for the current data structure.